fe/fe-core/src/main/java/com/starrocks/external/hive/events/AddPartitionEvent.java

```diff
@@ -105,6 +105,8 @@ protected boolean isSupported() {
     @Override
     protected void process() throws MetastoreNotificationException {
         if (!existInCache()) {
+            LOG.warn("Table [{}.{}.{}] doesn't exist in cache on event id: [{}]",
```

Comment:
Will this print too much log? And `info` level might be suitable.

Reply:
`warn` produces less logs.